### PR TITLE
Feature/view field description

### PR DESCRIPTION
**Pull request by dlesco**
Add schema attribute to views, and update documentation.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -124,6 +124,7 @@ resource "google_bigquery_table" "view" {
   dataset_id          = google_bigquery_dataset.main.dataset_id
   friendly_name       = each.key
   description         = each.value["description"]
+  schema              = lookup(each.value, "schema", null)
```
Comment (msgongora) [inline main.tf]: No need to do validation at the module level. This code should be enough.
```suggestion
  schema              = each.value["schema"]
```

File: README.md [hunk 2]
```diff
@@ -153,6 +153,7 @@ The `views` variable should be provided as a list of object with the following k
       billable = "true"
   }
   description = "A long description"                                 # Text describing the contents or purpose of the view.
+  schema = "<SCHEMA JSON DATA>"                                      # (Optional) JSON schema, to allow setting options such as field description
 }
 ```
```
Comment (msgongora) [inline README.md:159]: Also would be nice to update the [inputs](https://github.com/cbsi-dto/tf-mod-bq/blob/d0a4b7c804972a3bf48c803dca2538f00465c9de/README.md#inputs) from the readme.
https://github.com/cbsi-dto/tf-mod-bq/blob/f3afc9b296726cb9c5f4aa08808ea9a89df70987/README.md?plain=1#L201

**Review by msgongora — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (dlesco) [reply main.tf]: Changed, and tested here:
https://github.com/cbsi-dto/data-warehouse-ddl/pull/1138#issuecomment-1204466628

File: README.md (hunk 2 above)
Comment (dlesco) [reply README.md:159]: Updated this part of the docs

**Pull request merged**